yarn.lock

```diff
@@ -100,6 +140,27 @@
     "@babel/traverse" "^7.1.0"
     "@babel/types" "^7.0.0"
 
+"@babel/helper-call-delegate@^7.4.0":
```

Comment:
It looks like there are `^7.1.0` versions of a lot of these `^7.4.0` entries, which can be safely (or “safely”) upgraded to the 7.4.0.

Rather than perform surgery on yarn.lock you can straight delete the entries for "^7.1.0" dependencies and re-run `yarn install` and it’ll update those to `7.4.0` (unless `7.4.1` has snuck out in the interim).